pkg/formula/runner/inputs.go

```
@@ -246,3 +250,43 @@ func (in InputManager) resolveIfReserved(input formula.Input) (string, error) {
 	}
 	return "", nil
 }
+
+func (in InputManager) verifyConditional(cmd *exec.Cmd, input formula.Input) bool {
+	if input.Condition.Variable == "" {
+		return true
+	}
+
+	var value string
+	variable := input.Condition.Variable
+	for _, envVal := range cmd.Env {
+		components := strings.Split(envVal, "=")
+		if strings.ToLower(components[0]) == variable {
+			value = strings.ToLower(components[1])
+			break
+		}
+	}
+	if value == "" {
+		message := fmt.Sprintf("config.json: Conditional variable %s not found. Showing prompt item anyway.", variable)
```

You should return an error if the condition variable does not exist.
Consider the same reasons as the comment below.

---

You got it chief